### PR TITLE
suggestion for how to handle runtime conf

### PR DESCRIPTION
**Pull request by bryanhuntesl**

### CODE REVIEW

File: test/support/stub_config.ex [hunk 1]
```diff
@@ -0,0 +1,7 @@
+defmodule ReleaseAdmin.TestConfig do
```
Comment (sescobb27) [inline test/support/stub_config.ex]: what about moving this to the same folder as LiveConfig, so we don't have modules sparsed?

File: lib/release_admin/encryption/vault.ex [hunk 2]
```diff
@@ -1,13 +1,14 @@
 defmodule ReleaseAdmin.Encryption.Vault do
   use Cloak.Vault, otp_app: :my_app
 
-  alias ReleaseAdmin.Config
-
   @impl GenServer
   def init(config) do
+    secret_key =
+      :erlang.apply(Application.get_env(:release_admin, :runtime_config), :db_secret_key, [])
```
Comment (sescobb27) [inline lib/release_admin/encryption/vault.ex]: what about moving this line into `RuntimeConfig` so we don't pollute this module with env handling?

[the remark this answers is not in the document]
Comment (sescobb27) [reply lib/release_admin/encryption/vault.ex]: well maybe you can do something like
```ex
RuntimeCongfig.db_secret_key()

# then have something like

def db_secret_key do
  module = Application.get_env(:release_admin, :runtime_config)
  moduke.db_secret_key()
end
```

File: test/support/stub_config.ex (hunk 1 above)
Comment (bryanhuntesl) [reply test/support/stub_config.ex]: I was thinking if we shipped hex packages - we wouldn't be shipping the tests - make the download tiny bit smaller, also I've been putting anything that's stub into release_admin/test/support

**Pull request merged**